libsignal-protocol/src/macros.rs

```diff
@@ -1,13 +1,6 @@
 macro_rules! impl_serializable {
     ($name:ty, $serialize:ident, $deserialize:ident) => {
         impl $crate::Serializable for $name {
-            fn deserialize(_data: &[u8]) -> Result<Self, failure::Error>
```

Comment:
If the `Serializable` trait no longer takes care of `deserialize()`, should we add an `impl_deserializable!()` macro? Otherwise there won't be any way to deserialize anything from this crate that doesn't have a manual `Deserializable` impl.